### ISSUE TITLE
... (ellipsis) not used in t_summarytable

### ISSUE THREAD

**Issue by denisovan31415** [insightsengineering/goshawk#98]
```r
t_summarytable <- function(data,
                           trt_group,
                           param_var,
                           param,
                           xaxis_var,
                           facet_var = "AVISITCD",
                           loq_flag_var = "LOQFL", 
                           ...) {
```

The `...` is not used anyway. Should be removed along with the documentation

**Comment by donyunardi** [comment]
There are two modules in tmgoshawk that are using this function:

https://github.com/insightsengineering/teal.goshawk/blob/348f38eaecb68b170e3e47e583a3d6cae45cc566/R/tm_g_gh_density_distribution_plot.R#L398

https://github.com/insightsengineering/teal.goshawk/blob/348f38eaecb68b170e3e47e583a3d6cae45cc566/R/tm_g_gh_boxplot.R#L515

At first, I thought `...` is used for `tm_g_gh_density_distribution_plot` to capture the `font-size` argument, but looking at code base [here](https://github.com/insightsengineering/goshawk/blob/712a283a995d812f965bf163578f696a69a06b74/R/t_summarytable.R#L79), it doesn't look like the value passed in `...` are being executed in the function's body. I don't even see `...` was captured into a list.

which means, [this line](https://github.com/insightsengineering/teal.goshawk/blob/main/R/tm_g_gh_density_distribution_plot.R#L404) has no meaning to goshawk and has been working for tmgoshawk because of the `...`.

So, if we remove the `...` from `t_summarytable`, then we have to follow it by removing [this line](https://github.com/insightsengineering/teal.goshawk/blob/main/R/tm_g_gh_density_distribution_plot.R#L404).

Just to be safe, pinging @npaszty in case there's other concern.

**Comment by npaszty** [comment]
yes. t_summary used to present the tables below the visualizations

**Comment by donyunardi** [comment]
Thanks @npaszty.

In that case, we'll move forward with the removal of ellipsis from `t_summarytable ` since there's no use of it.

### Acceptance Criteria:
1. Remove `...` argument from the `t_summarytable` function in goshawk
2. Remove the `font-size` argument in [teal.goshawk::srv_g_density_distribution_plot](https://github.com/insightsengineering/teal.goshawk/blob/main/R/tm_g_gh_density_distribution_plot.R#L404)
3. Pass R CMD check and integration test.

These two has to be done (merged) at the same time otherwise the integration test will fail.

**Comment by npaszty** [comment]
yeah, I guess I can't provide much more info than that since I haven't looked at this code for a very long time.
